config/rbac/byohost_editor_clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: byohost-editor-clusterrole-binding
```

Comment:
nit
```suggestion
  name: byohost-editor-role-clusterrole-binding
  or
  name: byohost-editor-role-crb
```

Reply:
@pshail I tried to kept it close to the convention used by `kubebuilder` as in [here](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/config/rbac/role.yaml) and [here](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/config/rbac/role_binding.yaml).